pendingchanges/ax_bin.txt

```diff
@@ -0,0 +1,3 @@
+Improvements:
+ - Added a --thread [n] argument to the vdb_ax binary.
+ - Changed the positional arguments for the vdb_ax binary in execute mode. All positional arguments are now input grids which allows vdb_ax to read multiple files. Output files are specified with -o.
```

Comment:
do we need to make more of a big deal of the fact that this is changing, e.g. may need people to update any scripts using it?

Reply:
Oh I could rework this. Maybe we use `-i`? for example: 
`vdb_ax -i in1.vdb -i in2.vdb ...` 
`vdb_ax -i in1.vdb in2.vdb ...`
etc.  

Would prefer positional but maybe this is too breaking for now? We can then continue to support `in out` args with deprecation warning and transition it to an error.

Reply:
Would it be too unintuitive to do something like, require -i/-o and then if no file is input rather than execute do analyze --try-compile or something? This is probably tangential to this issue.

Reply:
Don't quite understand what you're suggesting

Reply:
I've added support for -i, currently works as `-i file.vdb -i file.vdb` etc. Added back support for `input.vdb output.vdb` but if this is used then -i/-o can't be used. e.g:

`input.vdb output.vdb` - works with a warning
`input.vdb` - works with a warning
`input.vdb -o output.vdb` - errors
`-i input.vdb output.vdb` - errors
`-i input.vdb -o output.vdb` - works, no warning